pkgs/applications/video/kodi-cli/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, makeWrapper, pkgs, curl, bash, jq, youtube-dl, gnome3, ...}:
+
+let
+  pkgName = "kodi-cli";
+  version = "1.1.1";
+in
+
+stdenv.mkDerivation rec {
+  name = pkgName + "-" + version;
```

Something more popular in nixpkgs would be
```suggestion
  name = "kodi-cli-${version}";
```

And move the `version = "1.1.1";` into the set doing away with the `let in`.